lib/erb-helper.coffee

```diff
@@ -23,6 +24,9 @@ module.exports =
   comment: ->
     @insertTag('<%#  %>')
 
+  end: ->
```

Comment:
Since the user will not have to add anything to the end tag we should add a switch to the `insertTag` method so it doesn't place the cursor inside the tag or surround the current selection